llvm/lib/Transforms/IPO/AttributorAttributes.cpp

```diff
@@ -11955,7 +11968,8 @@ struct AAUnderlyingObjectsImpl
     bool Changed = false;
     Changed |= DoUpdate(IntraAssumedUnderlyingObjects, AA::Intraprocedural);
     Changed |= DoUpdate(InterAssumedUnderlyingObjects, AA::Interprocedural);
-
+    if (!UsedAssumedInformation)
```

Comment:
I'm not sure if this is correct. Say, some `OtherAA` reach fixed point, but other `OtherAA` don't, can we still claim optimistic fixed point here?